libs/src/services/schemaValidator/schemas/trackSchema.json

```diff
@@ -114,6 +114,15 @@
                     "type": ["object", "null"],
                     "$ref": "#/definitions/RemixOf",
                     "default": null
+                },
+                "is_premium": {
+                    "type": "boolean",
+                    "default": false
+                },
+                "premium_conditions": {
```

Comment:
Is there another JSON schema in discovery node that needs updating?

Reply:
it has already been updated :)